templates/app/onboard-guide/index.html.twig

```diff
@@ -41,17 +41,9 @@
             </div>
         </div>
         <div class="slide fade-in slide-in">
-            <div class="header">
-                {# TODO: Get configutation to work so we dont need to comment out the below code #}
-                {# <h1>{{ get_configuration('app_terms_header')|default('Accept terms'|trans) }}</h1>
-                {% if get_configuration('app_terms_lead')|default(null) %}
-                    <p class="lead">{{ get_configuration('app_terms_lead') }}</p>
-                {% endif %} #}
-            </div>
```

Comment:
Oops, when I wrote “I think we can safely remove the comment.” I didn't actually mean “remove the comment”. I meant “uncomment the code”, i.e. remove `{#` and `#}`. We need to display the app terms.